app/build.gradle

```diff
@@ -32,11 +34,31 @@ dependencies {
 
     implementation 'com.squareup.retrofit2:retrofit:2.3.0'
```

Comment:
This is now duplicated with the newer version you add below, so it is redundant...